File: SampleSetup/Protocol/Model/AbstractModelWithId.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Model
{
    public abstract class AbstractModelWithId
    {
        public string Id;

        public string InsertUserName;
        public DateTime? InsertDate;

        public string UpdateUserName;
        public DateTime? UpdateDate;

        public bool RemoveIs;
        public string RemoveUserName;
        public DateTime? RemoveDate;
    }
}

File: SampleSetup/Output/PHP_8/Model/AbstractModelWithId.php
<?php
namespace SampleProject\Protocol\Model;

class AbstractModelWithId {
    public string $Id;

    public string $InsertUserName;
    public ?string $InsertDate;

    public string $UpdateUserName;
    public ?string $UpdateDate;

    public bool $RemoveIs;
    public string $RemoveUserName;
    public ?string $RemoveDate;
}
